Clinical-Genomics/scout
# Assign OMIM diagnoses to individuals or the entire case

- Assign individual OMIM diagnoses:
![image](https://user-images.githubusercontent.com/28093618/133042186-563edd6c-4a89-4b02-a719-6c16da9125ce.png)

- Improve OMIM terms description in ClinVar submission form:
![image](https://user-images.githubusercontent.com/28093618/133042134-a2dab8e7-ba03-4bc8-8ea5-589743ed7620.png)


This PR is slightly more complicated than the HPO on individuals one, because current case["diagnosis_phenotypes"] looks like this (a list of integers):

![image](https://user-images.githubusercontent.com/28093618/132643666-593c6099-5317-4af3-9c8a-cfd4630b1e9a.png)

To be able to assign OMIM terms to individuals, when a case is open (any case) the format of diagnosis_phenotypes is modified like this (reflects a similar structure of case HPO terms):

![image](https://user-images.githubusercontent.com/28093618/132645613-d85dba88-4c0d-456e-bbb6-fac4cc34a615.png)



**How to test**:
1. On master branch, clinical-db stage, assign one or more diagnoses to a case
2. Switch to this branch and make sure the case diagnoses are still visible (and can be removed)
3. Assign one more more diagnoses to one or more individuals of the case
4. Test that the new diagnoses can be removed from the case
5. Pin a variant and add it to a ClinVar submission. Make sure that OMIM diagnoses contain a description and not just the OMIM number in ClinVar submission.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
